Source/WebCore/css/parser/CSSParserToken.cpp

```diff
@@ -41,6 +41,29 @@
 namespace WebCore {
 DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(CSSParserToken);
 
+#ifdef NDEBUG
```

Comment:
Would be nice to get this check even in debug builds. Is there some reason that’s difficult?

Reply:
It failed on the debug bots and it was not immediately clear to me why but I can investigate.